### PR TITLE
Fix Duration type

### PR DESCRIPTION
**Pull request by pwltr**
- Fix Duration TS type

Small issue I came upon while building, Fairly sure this should match the Rust std::time::Duration trait.
https://github.com/rust-lang/rust/blob/03918badd33d255de806b4a9a8aa75b031ed0738/library/core/src/time.rs#L72-L75

### CODE REVIEW

File: src/stronghold.rs [hunk 1]
```diff
@@ -214,7 +214,7 @@ fn default_password_store() -> Arc<Mutex<HashMap<PathBuf, Arc<Password>>>> {
                 sleep(interval).await;
 
                 if interval.as_nanos() == 0 {
-                    continue;
+                    break;
```
Comment (lucasfernog) [inline src/stronghold.rs:217]: the password clear interval can still be changed later, so we can't break here (unless we start the process again with a new thread).

**Review by lucasfernog — CHANGES_REQUESTED** [review]
the current solution might fix the issue but it causes a problem if the clear interval is initially 0 and later changed to a different value.

**Comment by pwltr** [comment]
I see, well this may be over my head then. It works for my case for now. I will revisit this once I'm better at Rust :)

**Comment by pwltr** [comment]
Updated PR to only include changes of initial fix (Duration type)

**Comment by JonasKruckenberg** [comment]
See https://github.com/tauri-apps/plugins-workspace/pull/38

**Pull request closed without merging**